### DISCUSSION TITLE
Next ESMValTool Workshop: dates are fixed!

### DISCUSSION THREAD

**Discussion by hb326** [ESMValGroup/Community#141] [Meetings]
Hey all @ESMValGroup/esmvaltool-developmentteam,
thanks for everyone who had indicated their availability for the next ESMValTool workshop. We have a preferred date now:

**May 27 - May 29 2024**

Please block these dates in your calendars if you want to attend the workshop!
We decided to hold the workshop in **Germany at DLR (close to Munich)** again, so please check if you need any specific visa or something to come to the workshop. Let me (@hb326) know if you need an official invitation letter.

We will post more information in due course here (like accommotation options, travel information etc.).

**Comment by valeriupredoi** [comment]
![germany-bavaria](https://github.com/ESMValGroup/Community/assets/28983971/ece170ca-2df3-40b0-850d-86617f73c510)

**Comment by axel-lauer** [comment]
Please share your thoughts on topics you would like to discuss, ideas for sessions, meetings, breakout groups, etc, and wishes for coding sessions here, so we can start preparing an agenda for our workshop in May,

@ESMValGroup/esmvaltool-developmentteam

**Reply by valeriupredoi** [reply]
![schntzl](https://github.com/ESMValGroup/Community/assets/28983971/0604b058-c41c-400f-ac48-491676d874c7)

**Reply by malininae** [reply]
If there will be a chance, I'd like to give a presentation on how we operationalize ESMValTool for extreme event attribution and ask your opinion on how to make it better. Since it includes the recipes generated by python scripts, I think discussion on programmically generated recipes would be beneficial for a wider community. By then I will have a few PRs submitted on the core and on the tool, so general coding sessions on cleaning up PRs would be great.

**Reply by valeriupredoi** [reply]
Sounds great, a bit more meaty than my Schnitzel, Liza 😁

**Reply by schlunma** [reply]
Here are some topics I'd like to discuss:

- How can we read and process data from `intake` catalogs. Related issue: https://github.com/ESMValGroup/ESMValCore/issues/31.
- Re-organizing our configuration files. For example, this is relevant in order to use [different dask configurations](https://github.com/ESMValGroup/ESMValCore/issues/2369), to use [different DRS for different rootpaths](https://github.com/ESMValGroup/ESMValCore/issues/129), etc. Related issue: https://github.com/ESMValGroup/ESMValCore/issues/2371
- How can we read and process data in Zarr format.

**Reply by tepmo42** [reply]
**Here are the topics that were raised in the User Engagement Team meeting.** 

**Barriers to the ocean community using ESMVaTool:**
- They have there own tools that are often based on x-array rather than iris. 
- Challenges of dealing with ocean unstructured grids and tripolar grid in NEMO.  How difficult will it be to work with ocean grids in ESMValTool? 
- Less interested in CMIP comparisons, there is however interest in this from the coupled modelling community. 

**Cmorizing data is still a challenge for science users and can be a potential barrier:** 

- If metadata is wrong in a model (even if it cmorized) ESMValtool will crash. 

**Observation data management, access to observational data is important to facilitate using ESMValTool:**

 - There have been difficulties in keeping observation data updated across platforms. One solution to this could be that each site has its own cylc workflow to manage their own obs pool. 

Feedback from hackathon event is that **Jupyter notebooks** are a popular format for users, they  often find these easier to use than the standard recipe format.

**Reply by katjaweigel** [reply]
**High resolution model data**

- Often the weights to interpolate grids can be reused, not only for different variables (https://github.com/ESMValGroup/ESMValCore/pull/2344) but even across different experiments, time periods, and simulations of the same model. Could ESMValTool support the reading and use of precalculated weights because their calculation is very time consuming?
- Another vote to think about where it would make sense to use x-array.

(What) do we need to read model data on **HEALPix** grids? (often intake and Zarr, see @schlunma post above, but there might be more)

**Reply by ehogan** [reply]
A topic from the Met Office: an update on the upcoming (v2.11.0) release 😊

**Comment by rbeucher** [comment]
Thanks @hb326. I won't be able to join unfortunately but @flicj191 and @headmetal will be there.

**Comment by axel-lauer** [comment]
The next ESMValTool workshop will take place from **27 to 29 May 2024** in Oberpfaffenhofen (close to Munich), Germany. If you are interested in attending (online or in person), please fill out the registration form: https://forms.gle/7sDF5mL7BVP2DqoW6

Please don't forget to let us know about topics you would like to discuss, ideas for sessions, meetings, breakout groups, etc, and wishes for coding sessions, We will start drafting the agenda in early April.

**Comment by axel-lauer** [comment]
A first draft agenda for the next ESMValTool workshop is now available here: https://github.com/ESMValGroup/Community/discussions/162
Comments and feedback very welcome!
